.github/workflows/acc-tests.yaml

```diff
@@ -0,0 +1,69 @@
+# Flow based on proposal from: https://docs.google.com/document/d/1Gz0DohBnmH2WaqrQA3JCix_m6ISVyX56nYMDdT5worM/edit?usp=sharing
+
+# use like:
+
+# name: Run Acceptance Tests
+# on: push
+# jobs:
+#   run-acceptance-tests:
+#     uses: hashicorp/vault-workflows-common/.github/workflows/acc-tests.yaml@main
+#     secrets: inherit
+
+name: Run Acceptance Tests
+
+on:
+  workflow_call
+
+jobs:
+  run-acc-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        image:
+          - "hashicorp/vault:1.13"
+          - "hashicorp/vault:1.14"
+          - "hashicorp/vault:1.15"
+          - "hashicorp/vault:latest"
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: './go.mod'
+      - name: Build project
+        run: |
+          make dev
+      - name: Set up Vault server
```

Comment:
Maybe another future improvement, but it might be good to capture the Vault Server logs to help debug any failures

Reply:
Good call, added a step to output logs at the end of the run